### ISSUE TITLE
Keras version of MNC

### ISSUE THREAD

**Issue by McLawrence** [daijifeng001/MNC#62]
Hi. Is anybody aware of a Keras implementation of this architecture, or parts of it, like for example the ROI Warping layer or the proposal layer?

Thanks!

**Comment by MinaRe** [comment]
@McLawrence; have you found any repositories based on Keras o Tens?

**Comment by McLawrence** [comment]
@MinaRe I am sorry. I think I did not find one then. But there are implementations of faster R-CNN and Mask R-CNN in tensorflow or Keras.

**Comment by hgaiser** [comment]
I don't think there's a MNC implementation in Keras. There are plenty of MaskRCNN implementations though. I suggest using one of those as they are more accurate according to the paper.
